include/llzk-c/Dialect/Array.h

```diff
@@ -38,43 +41,31 @@ MLIR_DECLARE_CAPI_DIALECT_REGISTRATION(Array, llzk__array);
 
 /// Creates an llzk::array::ArrayType using a list of attributes as dimensions.
 MLIR_CAPI_EXPORTED MlirType
-llzkArrayTypeGet(MlirType type, intptr_t nDims, MlirAttribute const *dims);
-
-/// Returns true if the type is an llzk::array::ArrayType.
-LLZK_DECLARE_TYPE_ISA(ArrayType);
+llzkArrayArrayTypeGetWithDims(MlirType type, intptr_t nDims, MlirAttribute const *dims);
```

Comment:
Are we gonna do the nomenclature we talked about in #237 (`llzkArray_ArrayTypeGetWithDims`)?